frontend/src/pages/modelServing/screens/global/InferenceServiceListView.tsx

```diff
@@ -11,15 +10,20 @@ import ServeModelButton from './ServeModelButton';
 type InferenceServiceListViewProps = {
   inferenceServices: InferenceServiceKind[];
   servingRuntimes: ServingRuntimeKind[];
+  refreshInferenceServices: () => void;
+  refreshServingRuntimes: () => void;
+  refreshServerSecrets: () => void;
+  filterTokens: (servingRuntime?: string | undefined) => SecretKind[];
 };
 
 const InferenceServiceListView: React.FC<InferenceServiceListViewProps> = ({
   inferenceServices: unfilteredInferenceServices,
   servingRuntimes,
+  refreshInferenceServices,
+  refreshServingRuntimes,
+  refreshServerSecrets,
```

Comment:
This component doesn't use these individually, may as well accept a single `refresh` prop function and let the caller handle how to delegate to all three of these refresh functions.

Reply:
Yes good catch, I refactored the feature a little bit cause before it was working a little bit different and I didn't change this, thanks that's a great catch.